src/qt/pivx/dashboardwidget.cpp

```diff
@@ -205,6 +205,23 @@ void DashboardWidget::loadWalletModel(){
         filter->setSortRole(Qt::EditRole);
         filter->setSourceModel(txModel);
         filter->sort(TransactionTableModel::Date, Qt::DescendingOrder);
```

Comment:
Since `changeSort(nSortIndex)` is now called here in `loadWalletModel()` (and Date-DescendingOrder are the default values), isn't this initial/additional sorting `filter->sort(TransactionTableModel::Date, Qt::DescendingOrder)` redundant?